### ISSUE TITLE
ClusterID check can falsely produce errors

### ISSUE THREAD

**Issue by ysimonson** [pachyderm/pachyderm#4504]
**What happened?**: `pachctl` [now checks the ClusterID of the cluster it's communicating with](https://github.com/pachyderm/pachyderm/pull/4442) to ensure its not communicating with a different cluster than intended. In a few cases, I've now seen the issue where it's reporting an unexpected ClusterID when it shouldn't, producing errors like this:

```
error connecting to pachd: connected to the wrong cluster (context cluster ID = "f19b780a736c4286bc0ea929bb4e79a4" vs reported cluster ID = "8080955e8ac54bc19a67af3f7a33e781")
```

The dev reset script and and re-deploys will appropriately scrub the context's ClusterID, however there appears to be other cases where the ClusterID is not scrubbed. I'm not sure where yet.

**Workaround**: If you are currently facing this issue:

1) Open your pachyderm config file. By default it's at `~/.pachyderm/config.json`, but may be overridden via the `PACH_CONFIG` environment variable.

2) Remove the `cluster_id` entry in your active context's config.

**Comment by zeroidentidad** [comment]
In my case after deploying when I turned on the computer again and start the instance connected to minikube and verify with the **pachctl version** command I get:

> COMPONENT           VERSION             
> pachctl             1.9.11              
> **connected to the wrong cluster (context cluster ID = "56ae9a015d6740239c4e64618542f83d" vs reported cluster ID = "1be81be4db6c4642b13bbdfb28dbef7b")**

**Comment by zeroidentidad** [comment]
I see, thanks, doing that it works. removing the cluster_id entry in config.json

**Comment by ysimonson** [comment]
@zeroidentidad thanks for the details you added, it helped me track down the root cause!

**Comment by ysimonson** [comment]
This will be fixed in 1.10 and 1.9.12
